1.0-draft/index.html

```diff
@@ -618,21 +618,21 @@ <h3>Reconciliation Query Responses</h3>
         </p>
       </section>
       <section>
-        <h3>Sending Reconciliation Queries to a Service</h3>
+        <h3>Sending Reconciliation Queries to the Match Service</h3>
         <p>
-	  The primary role of a reconciliation service is to translate <a>reconciliation query batches</a> to
+	  The primary role of a reconciliation service is to provide a match service that translates <a>reconciliation query batches</a> to
```

Comment:
Maybe that would simpler (to avoid accumulating two nested "services" in the same sentence)?
```suggestion
	  The match service translates <a>reconciliation query batches</a> to
```